recipes/spades/3.5.0/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: spades
+  version: '3.5.0'
+
+source:
+  fn: SPAdes-3.5.0-Linux.tar.gz                                              # [linux]
+  url: http://spades.bioinf.spbau.ru/release3.5.0/SPAdes-3.5.0-Linux.tar.gz  # [linux]
+  md5: 1a4ef6e9c9da8894353fb76ed1a31c9d                                      # [linux]
+
+build:
+  number: 0
+  skip: True  # [py34 or py35 or py36 or osx]
```

Comment:
[py3k or osx] would be more correct isn't it?

Reply:
This version of SPAdes can be built with Python 3.3 (hence the original pinning), which does appear to be supported by conda. Using `[py3k or ...]` would presumably disable the package for Python 3.3 within bioconda.
Is there a neater way to maintain support for Python 3.3? Or should it just drop support for all Python3 versions? (It's not needed for the pipeline I'm trying to support.)

Reply:
We do not build for 3.3 anymore. See here: https://github.com/bioconda/bioconda-recipes/blob/master/scripts/env_matrix.yml#L1

Reply:
Thanks - it's as you say then!